README.md

```diff
@@ -11,11 +12,25 @@ Our relying parties can trust the collection of certificates because of the [GOV
 
 GOV.UK Verify expresses these anchors as [JSON Web Keys (JWK)](https://tools.ietf.org/html/rfc7517) and serves them signed in compact [JSON Web Signature (JWS)](https://tools.ietf.org/html/rfc7515) format.
 
+### About Connector Metadata Signer
+
+eIDAS specifies that we can only sign with RSASSA-PSS or ECDSA. Since our keys are RSA keys, we decided to go with the RSASSA-PSS algorithm. 
```

Comment:
I don't think we are using RSASSA-PSS anymore are we? (same goes for rest of README)

Reply:
yes!! great spot - i completely forgot about updating README